src/quick-lint-js/parse-expression-inl.h

```diff
@@ -1148,6 +1148,15 @@ expression* parser::parse_expression_remainder(Visitor& v, expression* ast,
 
   // f(x, y, z)  // Function call.
   case token_type::left_paren:
+
+    switch (ast->kind()) {
```

Comment:
Must fix: I think `ast` is wrong here. In most cases, `ast` is the beginning of the expression, not what is called. In the following example, `ast` refers to `x`, not `() => {}`:
```js
x - () => {}()
```
I think you want `binary_builder.last_expression()` instead.